apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/RegexUtils.java

```diff
@@ -0,0 +1,34 @@
+package gov.cms.bfd.server.war.commons;
+
+import com.google.common.collect.ImmutableList;
+
+/** Helper Utils class for Functions shared across. multiple classes. */
+public class RegexUtils {
+
+  /**
+   * Custom Split Function to replace.
+   *
+   * @param input a String that needs to remove whitespace and commas.
+   * @return Split Array.
+   */
+  public static String[] customSplit(String input) {
```

Comment:
The name should say what it's splitting on.  Maybe `splitCsv` or `splitOnCommas`?

Reply:
Changed to `splitOnCommas`